FirebaseUI/Auth/AuthProviderUI/Facebook/Source/FIRFacebookAuthUI.h

```diff
@@ -34,26 +34,26 @@ NS_ASSUME_NONNULL_BEGIN
     @brief The scopes to use with Facebook Login.
     @remarks Defaults to using "email" scopes.
  */
-@property(nonatomic, copy) NSArray<NSString *> *scopes;
+@property(nonatomic, copy, readonly) NSArray<NSString *> *scopes;
```

Comment:
Behavioral change. Over all I'd like to discuss how we want this sort of stuff to look like with the community. I personally want the login providers to be dumb, non-mutating objects that are basically data sources that don't actually have any behavior of their own. But we should discuss with others, since my perspective is lacking.